### PR TITLE
Very minor refactoring and edits to code comments

### PR DESCRIPTION
**Pull request by Garrett-R**
Just some miscellaneous improvements on things that tripped me up a bit.

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
@Garrett-R The changes to the docs and conf seem good to me.

For the clean-up in the code, I will let someone else review, as I am not familiar with that part of the code.

By the way, to make our (easier to review) and your (easier to update, quicker merged) life easier, it is often times a good approach to do separate PRs for clearly separate things (here it is not yet a big problem, as it are not that much of lines that are changed, but just giving the tip for possible future contributions)

**Comment by Garrett-R** [comment]
@jorisvandenbossche, right, good call.  I'll do that next time.

File: pandas/core/generic.py [hunk 1]
```diff
@@ -138,6 +138,9 @@ def _init_mgr(self, mgr, axes=None, dtype=None, copy=False):
 
     @property
     def _constructor(self):
+        """Used when a manipulation result has the same dimesions as the
+        original.
```
Comment (jreback) [inline pandas/core/generic.py]: more like: 

construct a class of this type with the same dimensions as the original

**Comment by jreback** [comment]
pls squash as well

File: pandas/core/generic.py (hunk 1 above)
Comment (Garrett-R) [inline pandas/core/generic.py]: I had actually just copied this docstring from [`internals.rst`](http://pandas.pydata.org/pandas-docs/stable/internals.html#override-constructor-properties).  Should I also the phrasing there too?

**Comment by jreback** [comment]
merged via https://github.com/pydata/pandas/commit/a7437430b5cb62e49a79b64d18eccfb2b4d6367f

thanks!

**Pull request closed without merging**